plugin.xml

```diff
@@ -288,6 +288,8 @@
             <string name="wifi_ssid_label">Network Name</string>
             <string name="wifi_type_label">Type</string>
         </config-file>
+
+        <dependency id="cordova-plugin-compat" version="^1.0.0" />
```

Comment:
cordova-android 5.0 can handle permissions itself, the cordova-plugin-compat allows to use plugins that require asking for permissions on any cordova-android version, even on versions older than 5.0. So, PermissionHelper is not part of cordova-android, but cordova-android 5.0 and greater have something similar.

So, PermissionHelper is not part of the core because it was created after cordova-android 5.0 to allow to use plugins in older versions as some people can't or don't want to update the platform